extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -30,32 +31,36 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 
 	private createAzureAccountsDropdown(view: azdata.ModelView): azdata.FormComponent {
 
-		this._azureAccountsDropdown = view.modelBuilder.dropDown().withValidation((c) => {
-			if ((<azdata.CategoryValue>c.value).displayName === constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR) {
-				this.wizard.message = {
-					text: constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR,
-					level: azdata.window.MessageLevel.Error
-				};
-				return false;
-			}
-			return true;
-		}).component();
+		this._azureAccountsDropdown = view.modelBuilder.dropDown()
```

Comment:
@alanrenmsft We should really consider adding Azure account/sub/RG dropdowns to the extension API. It's a pretty common control and having to re-implement it each time is not ideal.